vme/zone/professions.zon

```diff
@@ -238,29 +268,16 @@ var
    sl : stringlist;
 code
 {
-   if (ProfCount@professions(pc) == 1)
+   if (ProfCount@professions(pc) <= 2) 
```

Comment:
If this is a career check then I think this line should be:

if (... readyCareer(..)...) { ... }

It's totally fine to check "ProfCount" inside the { } if you want to do different messages.

But I think it's overall important to have just one logical function which determines if a new Career is possible or not.